wiki/FAQ/en.md

```diff
@@ -132,45 +128,44 @@ _Full explanation: [osu!taiko][osu!taiko wikilink]_
-##### Hit balloons
-They are shown as normal small red notes but with a inflatable balloon behind it.
--   You should hit the inner drum continuously until the hit counter reaches zero.
--   You gain points for each hit and bonus points for popping the balloon.
+If it is a small red note, you must press the button bound to the inner portion of the Taiko drum or hit the large flat area (center) of the TaTaCon. If it is a small blue note, you must press the button bound to the outer ring of the Taiko drum or hit the sides of the TaTaCon. If the note is a **LARGE** circle, you should press or hit both of either inner or outer drum depending on the colour of the note for additional points.
+
+The notes must be pressed or hit when they reach the small white circle next to the drum below the status bar on the playfield.
 
 ##### Drum rolls
-Drum rolls appears as yellow bar in the play-field.
--   You should hit the inner (or outer) drum continuously until it is done.
-  -   For LARGE note, you should hit both buttons on the inner (or outer) drum simultaneously and continuously until it is done.
--   You gain points for each successful hits.
--   You can choose to ignore it at the cost of your possible points gained from the drum roll.
+![Taiko drumroll][Taiko drumroll image]
+
+Drum rolls appears as yellow bar in the play-field with tiny hit-circles evenly-spaced inside.
+
+You should hit the inner (or outer) drum continuously for points until it is done. For **LARGE** note, you should hit both buttons on the inner (or outer) drum simultaneously and continuously until it is done. Do note that drum rolls hits were hardcapped and only counted when you hit on the tiny hitcircles, rather than drumrolling as fast as you can as in _Taiko no Tatsujin_. You can choose to ignore it with no health penalty but at the cost of your possible points gained from the drum roll.
 
 ##### Dendens
+![Taiko spinner][Taiko spinner image]
+
 These appear as spinners, with a countdown displaying the time you have remaining to clear it.
--   You must hit the inner and outer drum in order (like red,red, blue,blue, red,red, and so on) until the Denden meter is full.
--   Bonus points are given for extra hits after completing the Denden.
+
+You must hit the inner and outer drum in order (like red,red, blue,blue, red,red, and so on) until the Denden meter is full. Bonus points are given for extra hits after completing the Denden.
 
 #### Controls
 The default controls for the Taiko mod are:
 
-```
-Red: {Mouse} Left click, {Keyboard} [X] or [C], {TaTaCon} Flat surface of the drum.
-Blue: {Mouse} Right click, {Keyboard} [Z] or [V], {TaTaCon} Outer surface of the drum.
+Type | Mouse | Keyboard | TaTaCon
+---|---|---|---
+Red |Left click |[X] or [C]|Flat surface of the drum
+Blue|Right click|[Z] or [V]|Outer surface of the drum
 
 osu!cursor placement does not matter when playing.
-```
 
 You may configure the keyboard buttons as you see fit, from the Input tab in the Options menu.
```

Comment:
Added a section `How do I change the default keyboard key input for my game mode of choice?`